index.html

```diff
@@ -257,7 +257,7 @@ <h2>
       </p>
       <p>
         Algorithms defined in this specification assume that for each
-        <a>document</a> there is a <dfn>pending promise</dfn>, which is
+        <a>document</a> there is a <dfn>screen lock promise</dfn>, which is
```

Comment:
I think `<dfn>screen lock promise</dfn>` should actually be an internal slot, as per #144. The name for it should be: `<dfn>[[\promiseToLock]]</dfn>`.

Reply:
Names are hard... maybe promiseToLockScreen is better.